preferences/linux-legacy/kustomization.yaml

```diff
@@ -0,0 +1,26 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../base
+
+components:
+  - ./metadata
+  - ../components/interfacemodel-e1000e
+  - ../components/rng
```

Comment:
This adds a `virtio-rng-pci` device to the guest, can you remove this please?

Reply:
Done.